src/sentry/integrations/notifydisable.py

```diff
@@ -0,0 +1,56 @@
+from sentry.models import Organization
+from sentry.services.hybrid_cloud.integration import RpcIntegration
+
+provider_types = {
+    "integration": "integrations",
+    "plugin": "plugins",
+    "sentry-app": "sentry-apps",
+}
+
+
+def get_url(organization: Organization, provider_type: str, provider) -> str:
+    type_name = provider_types.get(provider_type, "")
+    return str(
+        organization.absolute_url(
+            f"/settings/{type_name}/{provider}/",
+        )
+    )
+
+
+def get_provider_type(redis_key) -> str:
+    for provider in provider_types:
+        if provider in redis_key:
+            return provider
+
+
+def get_subject(integration_name) -> str:
+    return f"Action required: re-authenticate or fix your {integration_name} integration"
+
+
+def notify_disable(
+    organization=Organization, integration=RpcIntegration, redis_key=str, project=None
```

Comment:
Should this be `organization: Organization` etc? Also I don't see `project` being used, does it need to be passed?

Reply:
I added project because I was thinking it would be used for when we get to plugins